concourse/pipelines/create-microbosh.yml

```diff
@@ -120,11 +122,14 @@ jobs:
           - -e
           - -c
           - |
-            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > bosh-secrets.yml
-            ls -l bosh-secrets.yml
+            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > generate/bosh-secrets.yml
+            ls -l generate/bosh-secrets.yml
+        outputs:
+        - name: generate
```

Comment:
At the other hand the behaviour before update was to implicitly create output from task with a task name assigned to it. Right now we are following this patter but we specify output name explicitly.